pkgs/tools/networking/openssh/default.nix

```diff
@@ -100,10 +103,12 @@ stdenv.mkDerivation rec {
   hardeningEnable = [ "pie" ];
 
   postInstall = ''
-    # Install ssh-copy-id, it's very useful.
-    cp contrib/ssh-copy-id $out/bin/
-    chmod +x $out/bin/ssh-copy-id
-    cp contrib/ssh-copy-id.1 $out/share/man/man1/
+    # Install ssh-copy-id, it's very useful. It's installed into a separate
+    # output so that macOS users wouldn't have to install the whole copy of ssh.
+    mkdir -p $ssh_copy_id/bin $man/share/man/man1
+    cp contrib/ssh-copy-id $ssh_copy_id/bin/
+    chmod +x $ssh_copy_id/bin/ssh-copy-id
+    cp contrib/ssh-copy-id.1 $man/share/man/man1/
```

Comment:
To avoid breaking things, can you install a symlink in $out as well? This is so that things like `lib.getBin openssh` have all of the outputs. This still allows for users to install ssh_copy_id output.

Something like:

```
$ ln -s $ssh_copy_id/bin/ssh-copy-id $out/bin/ssh-copy-id
```

Reply:
Fixed.